src/runtime_src/core/edge/drm/zocl/sched_exec.c

```diff
@@ -528,6 +529,9 @@ static irqreturn_t sched_exec_isr(int irq, void *arg)
 		return IRQ_NONE;
 	}
 
+	if(zocl_cu_get_control(&zdev->exec->zcu[cu_idx]) == AP_CTRL_CHAIN)
+		zocl_cu_check(&zdev->exec->zcu[cu_idx]);
```

Comment:
can we add some comment here to explain why we want o call cu_check in interrupt handler?